src/Microsoft.AspNetCore.Http.Abstractions/Extensions/HeaderDictionaryExtensions.cs

```diff
@@ -11,6 +11,7 @@ public static class HeaderDictionaryExtensions
         /// <summary>
         /// Add new values. Each item remains a separate array entry.
         /// </summary>
+        /// <param name="headers">The <see cref="IHeaderDictionary"/> that this method extends.</param>
```

Comment:
@ajaybhargavb yes but the docs for an extension method don't say "it's extending type Foo", rather, they say what the method does, i.e. "frobs a fribble using a frabulator."